src/screens/MetadataManagement.tsx

```diff
@@ -104,16 +103,10 @@ function MetadataManagement() {
     setApiResponse(json);
   };
   const handleCreateDatastream = async () => {
-    const res = await createDataStream(
-      authToken,
-      datastreamName,
-      parseInt(programId)
-    );
+    const res = await createDataStream(authToken, datastreamName);
 
     if (res.status != 200) {
-      setApiResponse(
-        "Bad request: programId id and datastream name are required"
-      );
+      setApiResponse("Bad request: datastream name is required");
```

Comment:
Couldn't there be other reasons why the response status was not 200 besides name not provided?

Reply:
This was purely for a POC, this page is no longer used and will be completely refactored down the line. Just had to update it to get it not bark at the new Interfaces and Types.